phoenix5-spark/README.md

```diff
@@ -118,15 +181,16 @@ val spark = SparkSession
 val df = spark.sqlContext
   .read
   .format("phoenix")
-  .options(Map("table" -> "INPUT_TABLE"))
+  .option("table", "INPUT_TABLE")
   .load
 
 // Save to OUTPUT_TABLE
 df.write
   .format("phoenix")
-  .mode(SaveMode.Overwrite)
-  .options(Map("table" -> "OUTPUT_TABLE"))
+  .mode(SaveMode.Append)
```

Comment:
Even if both work now, let's not change this in the example.
You may want to add a comment to indicate that either works now.

Reply:
Also use the other examples still use Overwrite.

Reply:
I made a mistake when duplicating examples, spark2 connector only support Overwrite mode, fixed.

Reply:
Thanks